.pre-commit-config.yaml

```diff
@@ -26,6 +26,10 @@ repos:
     hooks:
     -   id: codespell
         types_or: [python, rst, markdown]
+-   repo: https://github.com/MarcoGorelli/cython-lint
```

Comment:
just glancing at this, im stoked at the concept!

Reply:
does this have/need eyeballs?

Reply:
Would certainly help if you're interested, thanks!